contracts/wallet/AdminWallet.sol

```diff
@@ -101,6 +107,17 @@ contract AdminWallet is Ownable {
         emit AdminsRemoved(_admins);
     }
 
+    /* @dev top the first 50 admins
+     */
+    function topAdmins() public onlyOwner {
```

Comment:
maybe add as method parameter startIndex in case there are more than 50

Reply:
Have implemented this now